cms/djangoapps/models/settings/course_details.py

```diff
@@ -45,8 +45,8 @@ def __init__(self, org, course_id, run):
         self.enrollment_end = None
         self.syllabus = None  # a pdf file asset
         self.short_description = ""
-        self.overview = ""  # html to render as the overview
-        self.about_sidebar_html = "" # html to render as the about_sidebar_html
+        self.overview = ''  # html to render as the overview
```

Comment:
It still looks like this isn't a PEP8 fix but rather changing the quotes. Or am I missing something in the diff?

Assuming the former, let's revert this change so that this PR only touches lines with PEP8 violations.